core/src/test/kotlin/org/jetbrains/kotlinx/dataframe/io/json.kt

```diff
@@ -383,7 +410,7 @@ class JsonTests {
         ).alsoDebug("df:")
 
         val res = DataFrame.readJsonStr(df.toJson()).alsoDebug("res:")
-        res shouldBe df
+        res shouldBe df.convert { colsOf<Double?>() }.toFloat()
```

Comment:
I see that we are reducing Double -> Float what is possible, but why not Float -> Double as a most common type?

Reply:
There's no number unification happening in this test at all; in fact, we're not reducing Double to Float on purpose, however because we're writing a `Double` "1.0" and "3.0" to JSON it can be read back from JSON as `Float`. If any of the numbers were too large to fit in a `Float`,  or if the column contained both floats and integers, the result would have been `Double`.